Bottom: a1173dcb4cc4533a932dc2d424c6309277dcd37b
Top:    f5a41f4df632085615aea765d586075364b82f69
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 02:29:14 +0100

Refresh of hid-lesstif-process-all-pendin

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 08e2ad9..828fdac 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3928,6 +3928,9 @@ lesstif_progress (int so_far, int total, const char *message)
       XtDispatchEvent (&e);
     }
 
+  XtAppNextEvent (app_context, &e);
+  XtDispatchEvent (&e);
+
   /* Note the time we did this */
   gettimeofday (&last_time, NULL);
